dipy/align/streamwarp.py

```diff
@@ -73,7 +76,7 @@ def bundlewarp(static, moving, dist=None, alpha=0.3, beta=20, max_iter=15, affin
     alpha : float, optional
         Represents the trade-off between regularizing the deformation and
         having points match very closely. Lower value of alpha means high
-        deformations (default 0.3)
+        deformations (default 0.5)
```

Comment:
can you remove `(default 0.5)`. not needed in the docstring

Reply:
Should I remove all the default values for different parameters (beta, max_iter, etc.) from the docstring? @skoudoro

Reply:
Yes